content/api/commands/attachfile.md

```diff
@@ -0,0 +1,235 @@
+| `action`            | `'input'`                                                             | Switches attachFile between modes. Valid values are `input` and `drag-n-drop`. See [Actions](#Actions) below for more details.                                      |
+| `animationDistanceThreshold` | [`animationDistanceThreshold`](/guides/references/configuration#Actionability) | The distance in pixels an element must exceed over time to be [considered animating](/guides/core-concepts/interacting-with-elements#Animations). |
+| `force`             | `false`                                                               | Forces the action, disables [waiting for actionability](#Assertions).                                                                                               |
+| `log`               | `true`                                                                | Displays the command in the [Command log](/guides/core-concepts/test-runner#Command-Log).                                                                           |
+| `timeout`           | [`defaultCommandTimeout`](/guides/references/configuration#Timeouts)  | Time to wait for `.attachFile()` to resolve before [timing out](#Timeouts).                                                                                               |
+| `waitForAnimations` | [`waitForAnimations`](/guides/references/configuration#Actionability) | Whether to wait for elements to [finish animating](/guides/core-concepts/interacting-with-elements#Animations) before executing the command.                        |
+
+### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
+
+<List><li>`.attachFile()` yields the same subject it was given from the previous command.</li></List>
+
+### Action
+
+Depending on the action set in the `options` argument, `.attachFile()` can simulate two different user behaviors:
+
+#### `input` (default)
+
+By default, `attachFile()` runs in 'input' mode, mimicking a user selecting one or more files on an HTML5 input element. In this mode, the [subject](/guides/core-concepts/introduction-to-cypress#Subject-Management) must be a single `input` element with `type="file"`, or a `label` element connected to an input (either with its `for` attribute or by containing the input).
+
+#### `drag-n-drop`
+
+Setting the action to `drag-n-drop` changes the behavior of the command to instead mimic a user dragging files from the operating system into the browser, and dropping them over the selected subject. In this mode, the subject can be any DOM element or the `document` as a whole.
+
+## Examples
+
+### From a file on disk
+
+```javascript
+cy.get('input[type=file]').attachFile('file.json')
```

Comment:
It might be helpful to reinforce that the file can have a path

```suggestion
cy.get('input[type=file]').attachFile('path/to/file.json')
```

Reply:
Done.